static/app/views/issueDetails/groupFeatureFlags/groupFeatureFlagsDrawerContent.tsx

```diff
@@ -61,6 +65,18 @@ export default function GroupFeatureFlagsDrawerContent({
     return searchedTags;
   }, [data, search, tagValues]);
 
+  const organization = useOrganization();
+  const project = useProjectFromSlug({organization, projectSlug: group.project?.slug});
```

Comment:
does simply `group.project` not work for `project`?

```suggestion
  const project = group.project;
```

Reply:
Good point, I copied this pattern from another component, and I see in groupTagsDrawer we're also not directly using group.project. I wonder if this is intentional (group.project missing some data)?

Reply:
@roggenkemper @ryan953 was curious if you know the reason for not directly using group.proejct?

Reply:
types could be wrong. It could be that `group.project` might not always have all the fields present.

Reply:
Gotcha, I'll keep the useProjectFromSlug then